desktop/src/main.rs

```diff
@@ -67,6 +67,9 @@ fn init() {
     let subscriber = tracing_subscriber::fmt::Subscriber::builder()
         .with_env_filter(tracing_subscriber::EnvFilter::from_default_env())
         .finish();
+
+    env_logger::init();
```

Comment:
What's this for?

Reply:
This gives us logs from naga, which uses the `log` crate

Reply:
We already should be seeing those, we redirect log to tracing. This would replace that redirection

Reply:
It looks like we're only using `tracing_log` on web. I'll remove `env_logger` from this pr, and enable `tracing_log` in a separate PR.

Reply:
`tracing_subscriber` uses `tracing_log` by default, perhaps something's going wrong there? But I'm sure I've seen wgpu logs on desktop and those are log.